integration_tests/dbt_project.yml

```diff
@@ -22,9 +29,7 @@ seeds:
           recordkey: "{{ 'varchar(100)' if target.name in ('redshift', 'postgres') else 'string' }}"
           recordno: "{{ 'varchar(100)' if target.name in ('redshift', 'postgres') else 'string' }}"
           accountkey: "{{ 'int64' if target.name == 'bigquery' else 'bigint' }}"
-          accountno: "{{ 'varchar(100)' if target.name in ('redshift', 'postgres') else 'string' }}"
```

Comment:
No this was intentionally removed so we could recreate the same issue the customer experienced when the fields datatype didn't match. I would prefer we keep this change.

I didn't feel there was a need to callout this change since it doesn't impact the customer. However, we can include it for our own reference.

Reply:
Added CHANGELOG note